Kerdy/Kerdy/InitialSetting/VC/FirstInitialSettingVC.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  FirstInitialSettingVC.swift
+//  Kerdy
+//
+//  Created by 최다경 on 11/3/23.
+//
+
+import UIKit
+import SnapKit
+
+final class FirstInitialSettingVC: UIViewController {
+    
+    private lazy var progressLabel: UILabel = {
+        let label = UILabel()
+        label.text = "1/4"
+        label.font = .nanumSquare(to: .regular, size: 14)
+        label.textColor = .kerdyMain
+        return label
+    }()
+    
+    private lazy var nameSettingLabel: UILabel = {
+        let label = UILabel()
+        label.text = "이름 설정"
+        label.font = .nanumSquare(to: .regular, size: 14)
+        return label
+    }()
+    
+    private lazy var nameAskLabel: UILabel = {
+        let label = UILabel()
+        label.text = "이름이 무엇인가요?"
+        label.font = .nanumSquare(to: .regular, size: 20)
+        return label
+    }()
+    
+    private lazy var warningLabel: UILabel = {
+        let label = UILabel()
+        label.text = "설정 시 변경이 불가합니다."
+        label.font = .nanumSquare(to: .regular, size: 13)
+        label.textColor = .kerdyGray04
+        return label
+    }()
+    
+    private lazy var nameTextField: UITextField = {
+        let textField = UITextField()
+        textField.attributedPlaceholder = NSAttributedString(string: "입력해주세요.", attributes: [NSAttributedString.Key.foregroundColor : UIColor(named: "kerdy_gray01")!])
```

Comment:
forced unwrapping은 지양하는게 좋다고 생각합니다! 아래와 같이 옵셔널 바인딩을 활용하는건 어떨까요?
```suggestion
if let placeholderColor = UIColor.kerdyGray01 {
    let attributes: [NSAttributedString.Key : Any] = [NSAttributedString.Key.foregroundColor : placeholderColor]
    let attributedPlaceholder = NSAttributedString(string: "입력해주세요.", attributes: attributes)
    textField.attributedPlaceholder = attributedPlaceholder
} else {
    //  UIColor가 nil인 경우 에러 처리 또는 기본 색상 설정
}
```

Reply:
해당 방식으로 수정 시 에러가 발생해 forced unwrapping 제거하는 쪽으로 수정하겠습니다. 
감사합니다!